lxc/exec_windows.go

```diff
@@ -32,11 +34,24 @@ func (c *execCmd) getTERM() (string, bool) {
 }
 
 func (c *execCmd) controlSocketHandler(control *websocket.Conn) {
-	// TODO: figure out what the equivalent of signal.SIGWINCH is on
-	// windows and use that; for now if you resize your terminal it just
-	// won't work quite correctly.
-	err := c.sendTermSize(control)
-	if err != nil {
-		logger.Debugf("error setting term size %s", err)
+	ch := make(chan os.Signal, 10)
+	signal.Notify(ch, os.Interrupt)
```

Comment:
Make a channel that receives `os.Signal`'s and blocks if buffer of 10 entries is full.
I don't get what happens. Looks like `os.Interrupt` is send to `ch` to be immediately
catched below. I don't understand it - `ch` is not passed anywhere to get any other
kind of input.